go/vt/vttablet/tabletserver/tx_engine.go

```diff
@@ -106,7 +114,10 @@ func NewTxEngine(env tabletenv.Env, dxNotifier func()) *TxEngine {
 	te.txPool = NewTxPool(env, limiter)
 	// We initially allow twoPC (handles vttablet restarts).
 	// We will disallow them, when a new tablet is promoted if semi-sync is turned off.
```

Comment:
I think we should update the comments here too.

Reply:
Done